.github/workflows/acceptance-tests-runner.yml

```diff
@@ -233,9 +233,11 @@ jobs:
       with:
         filters: |
           advanced_cluster:
-            - 'internal/service/advancedcluster/*.go'
+            - 'internal/service/advancedcluster/!(*_test).go' # matches any adv_cluster file change except test files  
+
```

Comment:
nit: delete empty line

Reply:
done in df97d850eb76f8988014320c3c0e40f3f2fd521c